portal/src/components/Materials/Materials.component.js

```diff
@@ -52,23 +55,12 @@ export default {
     },
     /**
      * Select material from collection
```

Comment:
Nu is die material param ook weg. Wat mij betreft mogen die hele comments weg, want het geeft mij alleen maar ruis. Ik zie ook wel dat het een material select aan de naam en dat het een material als argument verwacht (ik weet dat het er al zo stond 😉 ).

Reply:
ok